console/atest-ui/src/views/TestCase.vue

```diff
@@ -601,8 +602,9 @@ const duplicateTestCase = () => {
 
         <el-tab-pane name="header">
           <template #label>
-            <el-badge :value="testCaseWithSuite.data.request.header.length - 1"
-              :hidden="testCaseWithSuite.data.request.header.length === 1" class="item">Header</el-badge>
```

Comment:
I didn't try it. But I'm wondering if below code can work well.

```vue
:hidden="testCaseWithSuite.data.request.header.length <= 1"
```

Reply:
yes, we can use <= 1, here is screenshot:
![demo](https://github.com/LinuxSuRen/api-testing/assets/98792230/b8de1775-2988-443c-9e97-c594c57495e7)

Reply:
Now badge 0 will not appear

Reply:
Great.